dxr/static/js/dxr.js

```diff
@@ -140,7 +140,7 @@ $(function() {
         requestsInFlight = 0,  // Number of search requests in flight, so we know whether to hide the activity indicator
         displayedRequestNumber = 0,
         didScroll = false,
-        resultCount = 0,
+        resultsLineCount = 0,
```

Comment:
From what I can tell, `previousDataLimit` is the amount of lines returned by the last request to the API (which is determined by the amount of lines we can show on a single screen, I think). If `resultsLineCount` is equal to `previousDataLimit` then presumably we got as many lines as we requested from the API, and there may be more (or we might be "lucky" and asked for just the right amount and hit the end of the list.

So it's used to determine if we need to fetch any more results when an infinite scroll request is triggered. I don't think it's the best way to determine that but I also just want to make it work for now, and refactor it down later.

In a related note, I'm not entirely sure the math is working out exactly right anyway, as locally infinite scroll seems to trigger too early, but I couldn't see where the issue was and figured it wasn't a blocker as it doesn't affect functionality much.